udata/core/dataset/rdf.py

```diff
@@ -475,6 +475,30 @@ def access_rights_from_rdf(resource: RdfResource):
         return rdf_value(top_el, RDFS.label) or rdf_value(resource, DCT.accessRights)
 
 
+def licenses_from_rdf(resource: RdfResource) -> set[str]:
+    """
+    Extract licences from a RDF distribution.
+
+    Will match at least the following:
+    +   <dct:license>
+            <dct:LicenseDocument>
+                <rdfs:label xml:lang="fr">license</rdfs:label>
+            </dct:LicenseDocument>
+        </dct:license>
+    +   <dct:license>license</dct:license>
+    """
+    licenses = set()
+    for predicate in DCT.license, DCT.rights:
+        for license_info in resource.objects(predicate=predicate):
+            if isinstance(license_info, (URIRef, Literal)):
+                licenses.add(license_info.toPython())
+            elif isinstance(license_info, RdfResource):
```

Comment:
`value.identifier.toPython()` not relevant anymore? I'm not sure what it did, so open question...

Reply:
Looks like a safe way to serialize, probably not much over `str`? But it's used all over the place, like in `rdf_value`, so I'll keep it as best practice.